submissions/antonkottans/memory-pair-game/script.js

```diff
@@ -0,0 +1,250 @@
+const pipe = (...arrayOfFunctions) => {
+    if (arrayOfFunctions.length === 0) return undefined;
+    else if (arrayOfFunctions.length === 1) return arrayOfFunctions[0]();
+    return arrayOfFunctions.reduce((prev, func) => {
+        return func(prev);
+    }, undefined);
+};
+let timerId = null;
+
+const state = {
+    timerId: null,
+    guessedPairsAmount: 0,
+    flipped: false,
+    guessedPairs: [],
+    guessedIds: new Set(),
```

Comment:
Why `guessedPairds` and `cardContainers` are arrays and `guessedIds` and `notGuessedIds` are sets?